tests/test_nepc.py

```diff
@@ -84,6 +84,16 @@ def test_CS_class(nepc_connect):
     assert isinstance(cs.data["sigma"], list)
     assert isinstance(cs.data["sigma"][0], float)
     # test len()
+    assert(len(cs) == len(cs.data['e']))
```

Comment:
there is already a test for `len` below that uses external knowledge of the number of data points. I don't think we want to have unit tests that use the same methods in the assertion that are used in the functions being tested.